manifests/ha/base/redis-ha/chart/upstream.yaml

```diff
@@ -701,6 +701,10 @@ data:
       stats enable
       stats uri /stats
       stats refresh 10s
+    # Additional configuration
+    global
+      maxconn 4096
```

Comment:
Is the value already being used by some code?

Reply:
My understanding here is this value used to be based on the max file descriptors the kernel would give you, in RHEL7 and 8 I believe this used to be 4096 (hence me using this value) in RHEL9 and i believe some other distro's this limit is removed and ha-proxy just tries to use all the file descriptors on the box until it runs out of memory now unless you set maxconn in it's config. 

I don't see any harm in reducing the number but this was just chosen to be the absolute worst-case situation to maintain functionality